pages/linux/nmcli-general.md

```diff
@@ -0,0 +1,29 @@
+# nmcli general
+
+> Manage general settings of NetworkManager.
+> This subcommand can also be called with `nmcli g`.
+> More information: <https://networkmanager.dev/docs/api/latest/nmcli.html>.
+
+- Show the general status of NetworkManager:
+
+`nmcli general`
+
+- Show the hostname of the current device:
+
+`nmcli general hostname`
+
+- Change the hostname of the current device:
+
+`sudo nmcli general hostname {{new_hostname}}`
+
+- Show the permissions of NetworkManager:
```

Comment:
```suggestion
- Show permissions of NetworkManager:
```

Reply:
Why should we remove `the`? It's a lot less formal without.

Reply:
Should we write formal tldr pages or not?